iOS/SonarKit/CppBridge/SonarCppBridgingConnection.mm

```diff
@@ -6,9 +6,7 @@
  *
  */
 #import "SonarCppBridgingConnection.h"
-
-#import <FBCxxUtils/FBCxxFollyDynamicConvert.h>
```

Comment:
This changes will break internally for us. You can either give the path like Pods/SonarKit/iOS/SonarKit or create a podspec for the same. I think creating podscpec for FBCxxUtils will be the good way around.